code/game/machinery/computer/records/records.dm

```diff
@@ -43,19 +43,24 @@
 
 	switch(action)
 		if("edit_field")
-			target_record = locate(params["record_ref"]) in GLOB.manifest.general
+			if (!target_record)
+				return
+			if (!authenticated)
+				return FALSE
 			var/field = params["field"]
-			if(!field || !(field in target_record?.vars))
+			if(!field || !can_edit_field(field))
```

Comment:
There are no fields that can be editted in the override bellow

Reply:
It gets overriden for medical and security consoles. This console is abstract